imports/plugins/core/versions/index.js

```diff
@@ -0,0 +1 @@
+export { Migrations as Migrations } from "meteor/percolate:migrations";
```

Comment:
This can be:
```js
export { Migrations } from "meteor/percolate:migrations";
```

Certainly nothing wrong with it.  Just trying to be consistent with syntax.